src/main/java/com/puppycrawl/tools/checkstyle/AbstractAutomaticBean.java

```diff
@@ -84,6 +85,9 @@ public enum OutputStreamOptions {
     /** Comma separator for StringTokenizer. */
     private static final String COMMA_SEPARATOR = ",";
 
+    /** Separator that splits multiple regex expression in a string. */
+    private static final String REGEX_SEPARATOR = COMMA_SEPARATOR;
```

Comment:
I don't see the need for a field to exactly duplicate another field. Just remove it and use `COMMA_SEPARATOR`.

Reply:
done